awx/main/utils/licensing.py

```diff
@@ -383,12 +383,20 @@ def validate(self):
             current_instances = Host.objects.active_count()
         else:
             current_instances = 0
+        license_date = int(attrs.get('license_date', 0) or 0)
+        automated_instances = HostMetric.objects.count()
+        first_host = HostMetric.objects.only('first_automation').order_by('first_automation').first()
+        if first_host:
+            automated_since = int(first_host.first_automation.timestamp())
+        else:
+            automated_since = int(Instance.objects.order_by('id').first().created.timestamp())
```

Comment:
The prior version of this used `license_date` for the since date if there are no automated hosts... which is the license *end* date. Oops.

This uses the nearest analog I can think of for "install date". @chrismeyersfsu do you see any problems with this?

Reply:
Instance created date is better than I would have thought.

Below is from our dogfood server that has been alive for a long time (has migrated servers and everything).
![image](https://user-images.githubusercontent.com/722880/121182115-c2602b80-c830-11eb-802a-3a6743d1aa8f.png)

vs.

![image](https://user-images.githubusercontent.com/722880/121182613-474b4500-c831-11eb-94b0-e82dd6f54abd.png)

 `int(Instance.objects.order_by('created').first().created.timestamp())` <-- maybe find the oldest `Instance`. Id is _pretty_ much that so it's fine as is.